docker/scripts/get_templates.sh

```diff
@@ -8,6 +8,8 @@ OASIS_TEMPLATE="https://files.osf.io/v1/resources/fvuh8/providers/osfstorage/584
 OASIS_SHA256="d87300e91346c16f55baf6f54f5f990bc020b61e8d5df9bcc3abb0cc4b943113"
 NKI_TEMPLATE="https://files.osf.io/v1/resources/fvuh8/providers/osfstorage/59cd90f46c613b02b3d79782"
 NKI_SHA256="4bba067f6675d15be96b205cb227e18a540673fd7e4577e13feedcef3a6f0ec5"
+OASIS_DTK31_TEMPLATE="https://files.osf.io/v1/resources/fvuh8/providers/osfstorage/5b16f17aeca4a80012bd7542"
```

Comment:
You use `OASIS_DTK31_TEMPLATE` here, but `$OASIS_DTK31_ATLAS` below. Either one is fine with me.

Reply:
Thanks.
Fixed now.